keps/sig-apps/3335-statefulset-slice/kep.yaml

```diff
@@ -15,26 +15,26 @@ approvers:
   - "@soltysh"
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: alpha
+stage: beta
 
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.26"
+latest-milestone: "v1.27"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
   alpha: "v1.26"
+  beta: "v1.27"
 
 # The following PRR answers are required at alpha release
 # List the feature gate name and the components for which it must be enabled
 feature-gates:
-  - name: StatefulSetSlice
+  - name: StatefulSetStartOrdinal
     components:
       - kube-apiserver
       - kube-controller-manager
 disable-supported: true
 
 # The following PRR answers are required at beta release
-# metrics:
-# Currently no metrics are planned for alpha release.
+metrics:
```

Comment:
I'd suggest considering a metric which would help with monitoring this functionality.

Reply:
Added a metric that tracks if the user specifies the `.spec.ordinals.start` field: `kube_statefulset_ordinals_start`.